### ISSUE TITLE
Module with Git source crash - 0.13 beta1

### ISSUE THREAD

**Issue by kevinmarx** [hashicorp/terraform#25264]
### Terraform Version

```
0.13beta1
```

### Terraform Configuration Files

```hcl
module "my_module" {
  source = "git::git@github.com:flipgrid/tf_modules.git//my_module"
}
```

### Crash Output

https://gist.github.com/kevinmarx/35e73f4e78032af547e4fbe0709ce8d3
### Steps to Reproduce

1. `module source as Github repo with path`
2. `terraform init`

**Comment by danieldreier** [comment]
@kevinmarx thanks for reporting this. I don't have access to the module you're using, so I created a new trivial module example to reproduce this, but was unsuccessful. The reproduction case I made is in https://github.com/danieldreier/terraform-issue-reproductions/tree/master/25264 which references https://github.com/danieldreier/example-terraform-module

The crash output you're showing is pretty clear that there is a real issue here, but the reproduction steps aren't quite enough to reproduce it. I believe you're having a real issue and would like to reproduce it. 

Can you please make a PR to update the reproduction case in https://github.com/danieldreier/terraform-issue-reproductions/tree/master/25264 so that it reproduces the problem you're describing?

**Comment by kevinmarx** [comment]
Sure thing, I'll see if I can reproduce by creating a module similar to the one we're using 👍

**Comment by jbardin** [comment]
Hi @kevinmarx,

This should be fixed in the master branch, and is a dupe of #25177.

Thanks !
